include/string_utils.h

```diff
@@ -9,14 +9,14 @@
 
 struct StringUtils {
     UErrorCode status;
-    icu::Transliterator* transliterator;
+    //icu::Transliterator* transliterator;
 
-    StringUtils():status(U_ZERO_ERROR),
-                  transliterator(icu::Transliterator::createInstance("Latin-ASCII", UTRANS_FORWARD, status)) {
+    StringUtils(): status(U_ZERO_ERROR) {
+        // transliterator(icu::Transliterator::createInstance("Latin-ASCII", UTRANS_FORWARD, status))
```

Comment:
Did you intend to leave this line commented out?

Reply:
Yes, this was dead code, but I wanted to leave it around as it might be something that would be revisited in near future and I took some time to figure out that `Latin-ASCII` format.